pallets/interest-accrual/src/migrations.rs

```diff
@@ -110,6 +110,23 @@ pub mod v2 {
 	}
 }
 
+pub mod centrifuge {
+	use super::*;
+
+	pub struct SetStorageVersionToV2<T>(sp_std::marker::PhantomData<T>);
+
+	impl<T: Config> OnRuntimeUpgrade for SetStorageVersionToV2<T> {
```

Comment:
In theory we could change `Release::default()` to be V2 now, and the unpopulated storage key would be V2 on Centrifuge without us needing to run a migration

Reply:
Removed and defaulted to V2.